packages/cli/test/integration.test.ts

```diff
@@ -36,6 +35,10 @@ function execa(file, args, options) {
   return _execa(file, args, options);
 }
 
+function expectFile(path) {
+  expect(path, `file should exist at "${path}"`).toBe(true);
+}
```

Comment:
Usually these helpers actually make the test failures harder to read because the line number will say 39 instead of the actual line that failed

Reply:
That's a good point. I'll inline this.